backend/src/main/resources/application.properties

```diff
@@ -1,8 +1,11 @@
+spring.sql.init.mode=always
```

Comment:
이 부분의 설정은 어떤걸까요?!

Reply:
Spring Boot 2.5 이후 `schema.sql` 및 `data.sql` 스크립트를 동작시키기 위한 용도로 사용하는 설정입니다. 아래는 Spring Boot 2.5 릴리즈 노트를 번역한 것입니다.

### **`schema.sql` and `data.sql` Files**

With Spring Boot 2.5.1 and above, the new SQL initialization properties support detection of embedded datasources for JDBC and R2DBC. By default, SQL database initialization is only performed when using an embedded in-memory database. To always initialize a SQL database, irrespective of its type, set `spring.sql.init.mode` to `always`. Similarly, to disable initialization, set `spring.sql.init.mode` to `never`.

Spring Boot 2.5.1 이상에서는 새로운 SQL 초기화 속성이 JDBC 및 R2DBC에 대한 embedded datasource 탐지를 지원한다. 기본적으로 SQL database 초기화는 embedded in-memory database를 사용하는 경우에만 수행된다. SQL database를 항상 초기화 하려면 데이터베이스 유형에 관계없이 `spring.sql.init.mode`를 `always`로 설정한다. 마찬가지로 초기화를 사용하지 않으려면 `spring.sql.init.mode`를 `never`로 설정한다.

### **Hibernate and data.sql**

By default, `data.sql` scripts are now run before Hibernate is initialized. This aligns the behavior of basic script-based initialization with that of Flyway and Liquibase. If you want to use `data.sql` to populate a schema created by Hibernate, set `spring.jpa.defer-datasource-initialization` to `true`. While mixing database initialization technologies is not recommended, this will also allow you to use a `schema.sql` script to build upon a Hibernate-created schema before it’s populated via `data.sql`.

기본적으로 `data.sql` 스크립트는 hibernate가 초기화되기 전에 실행된다. 이는 기본 스크립트 기반 초기화의 동작을 Flyway와 Liqubase의 초기화와 일치 시킨다. 만약 `data.sql`을 사용하여 Hibernate에서 생성한 스키마를 채우려면 `spring.jpa.defer-datasource-initialization`를 true로 설정한다. 데이터베이스 초기화 기술을 혼합하는 것은 권장하지 않지만, `data.sql`을 통해 채워지기 전에 `schema.sql` 스크립트를 사용하여 Hibernate가 생성한 스키마를 빌드할 수도 있다.